templates/certified/desktops.html

```diff
@@ -0,0 +1,189 @@
+                    </label>
+                    {% endfor %}
+                  </section>
+                  <span class="js-toggle-links">
+                    <p class="p-reveal-versions is-dense"><a> Show all versions ({{release_filters|length - 5}})</a></p>
+                    <p class="p-reveal-versions is-dense u-hide"><a>Show fewer</a></p>
+                  </span>
+                </li>
+              </ul>
+            </aside>
+            <button class="p-button--positive p-update-results js-apply-filters" type="submit" disabled>Apply filters</button>
+            <button class="p-button--neutral p-update-results" onclick="return clearFilters()">Clear filters</button>
+          </div>
+        </nav>
+      </div>
+      {% if total_results > 0 %}
+      <div class="col-9">
+        <table class="p-certification-results">
+          <thead>
+            <tr>
+              <th><span class="p-certification-header">Vendor</span></th>
+              <th><span class="p-certification-header">Model</span></th>
+              <th><span class="p-certification-header">Category</span></th>
+            </tr>
+          </thead>
+          <tbody>
+            {% for result in results %}
+            <tr>
+              <td><a href="/certified?vendor={{ result.make }}">{{ result.make }}</a></td>
+              <td><a href="/certified/{{ result.canonical_id }}">{{ result.make }} {{ result.model }}</a></td>
+              <td>{{ result.category }}</td>
+            </tr>
+            {% endfor %}
+          </tbody>
+        </table>
+        <div class="row">
+          <div class="col-5">
+            {% with total_results=total_results, total_pages=total_pages, offset=offset, limit=limit, align="u-align--left"%}
+            {% include "security/cve/_pagination.html" %}
+            {% endwith %}
+          </div>
+          <div class="col-4 u-align--right">
+            {% if total_results > 0 %}
+            <p style="display: inline-block; padding-right: 0.5rem;">Results per page</p>
+            <select class="p-results-per-page" name="limit" aria-label="Results per page" style="max-width: 5.3rem; min-width: 5.3rem;" onchange=updateResultsPerPage() data-limit="{{ limit }}">
+              <option value="20" {% if limit == 20 %}selected{% endif %}>20</option>
+              {% if total_results > 20 %}
+              <option value="40" {% if limit == 40 %}selected{% endif %}>40</option>
+              {% endif %}
+              {% if total_results > 40 %}
+              <option value="60" {% if limit == 60 %}selected{% endif %}>60</option>
+              {% endif %}
+              <option value="{{ total_results }}" name="all">All</option>
+            </select>
+            {% endif %}
+          </div>
+        </div>
+      </div>
+      {% else %}
+      <div class="col-9">
+        <h3 class="p-heading--4">No results - why not try widening your search?</h3>
```

Comment:
Maybe "Sorry we found on results matching your criteria. Please try and widen your search."